File: GiletaProjectNotIOS/app/src/main/java/com/example/taskapp/model/Task.kt
package com.example.taskapp.model

import java.util.*

data class Task(
    val id: UUID = UUID.randomUUID(),
    val title: String,
    val description: String? = null,
    val priority: Int = 1,
    val flag: Boolean = false,
    val dueDate: Date? = null,
    var isCompleted: Boolean = false
)

File: GiletaProject/GiletaProject/Task.swift
//
//  Task.swift
//  GiletaProject
//
//  Created by Alexander on 30.11.2025.
//
import UIKit

struct Task {
    var title: String
    var description: String?
    var priority: Int 
    var flagged: Bool
    var deadline: Date?
    var isCompleted: Bool = false
}
